frontend/src/features/workspace/components/WorkspaceSearchbar/MobileWorkspaceSearchbar.tsx

```diff
@@ -0,0 +1,165 @@
+import { useCallback, useMemo } from 'react'
+import { BiFilter, BiSearch, BiX } from 'react-icons/bi'
+import {
+  Box,
+  Flex,
+  forwardRef,
+  Icon,
+  Input,
+  InputGroup,
+  InputLeftElement,
+  InputRightElement,
+  MenuButton,
+  MenuItemOption,
+  MenuOptionGroup,
+  Portal,
+} from '@chakra-ui/react'
+
+import IconButton from '~components/IconButton'
+import Menu from '~components/Menu'
+
+import { FilterOption } from '~features/workspace/types'
+import { FILTER_OPTIONS } from '~features/workspace/utils/dashboardFilter'
+
+import {
+  useWorkspaceSearchbar,
+  WorkspaceSearchbarProps,
+} from './WorkspaceSearchbar'
+
+export interface MobileWorkspaceSearchbarProps extends WorkspaceSearchbarProps {
+  isExpanded: boolean
+  onToggleExpansion: () => void
+}
+
+const SelectedFilterAffordance = ({ show }: { show?: boolean }) => {
+  return (
+    <>
+      <BiFilter fontSize="1.25rem" />
+      {show && (
+        <Flex
+          aria-hidden
+          width="1rem"
+          height="1rem"
+          as="span"
+          color="white"
+          position="absolute"
+          top="-0.5rem"
+          right="-0.5rem"
+          textStyle="legal"
+          bg="primary.500"
+          borderRadius="50%"
+          zIndex="banner"
+          align="center"
+          justify="center"
+        >
+          1
+        </Flex>
+      )}
+    </>
+  )
+}
+
+export const MobileWorkspaceSearchbar = forwardRef<
+  MobileWorkspaceSearchbarProps,
+  'input'
+>(
+  (
+    { isExpanded, onToggleExpansion, placeholder, ...props },
+    ref,
+  ): JSX.Element => {
+    const {
+      internalFilter,
+      internalValue,
+      setInternalFilter,
+      setInternalValue,
+    } = useWorkspaceSearchbar(props)
+
+    const hasFilter = useMemo(
+      () => internalFilter !== FilterOption.AllForms,
+      [internalFilter],
+    )
+
+    const handleToggle = useCallback(() => {
+      if (isExpanded) {
+        setInternalValue('')
+      }
+      onToggleExpansion()
+    }, [isExpanded, onToggleExpansion, setInternalValue])
+
+    return (
+      <>
+        <Box gridArea="searchicon">
```

Comment:
tiny nit and might not be worth it, but I wonder if the `gridArea` names should be in the props somehow. currently the styling of this component and its parent are coupled through the grid area names, might be hard to debug in future. but also might make the props messy so up to you

Reply:
Personally I think it'll make the props a bit messy but i agree we should do something to make the coupling more explicit. Will add comments to address this